extensions/resource-deployment/src/ui/deployAzureSQLDBWizard/deployAzureSQLDBWizard.ts

```diff
@@ -47,14 +53,25 @@ export class DeployAzureSQLDBWizard extends WizardBase<DeployAzureSQLDBWizard, W
 	}
 
 	protected async onOk(): Promise<void> {
-		await this.scriptToNotebook();
+		if (instanceOfAzureSQLDBDeploymentProvider(this.resourceProvider)) {
+			await this.scriptToNotebook();
+		} else {
+			super.onOk();
+		}
+	}
+
+	public refreshWizard() {
+		if (instanceOfAzureSQLDBDeploymentProvider(this.resourceProvider)) {
+			this._wizardInfo = this.resourceProvider.azureSQLDBWizard;
+		}
 	}
 
 	protected onCancel(): void {
 	}
 
-	private getPages(): WizardPageBase<DeployAzureSQLDBWizard>[] {
-		const pages: WizardPageBase<DeployAzureSQLDBWizard>[] = [];
+	private getPages(): WizardPageBase<DeployAzureSQLDBWizard, DeployAzureSQLDBWizardModel>[] {
+		const pages: WizardPageBase<DeployAzureSQLDBWizard, DeployAzureSQLDBWizardModel>[] = [];
```

Comment:
Declare declare it all in one - like 
```
const pages: WizardPageBase<DeployAzureSQLDBWizard, DeployAzureSQLDBWizardModel>[] = [
   new ToolsAndEulaPage(...),
   new AzureSettingsPage(tihs),
   ...
];
```

Reply:
@aasimkhan30 It's minor but it'd be cleaner to do it that way - any reason you aren't?